common/src/commonMain/sqldelight/io/xorum/codeforceswatcher/1.sqm

```diff
@@ -0,0 +1 @@
+CREATE TABLE Contest (id INTEGER NOT NULL, name TEXT NOT NULL, time INTEGER NOT NULL, phase TEXT NOT NULL, PRIMARY KEY(id))
```

Comment:
It migrations doesn't work in the current version, it's better to remove them.

Reply:
Are you sure?

Reply:
I wanted to say `if migrations`. But yes, I'm almost sure these migrations aren't needed. Can you check?

Reply:
They work, otherwise, how I install application to old version db and everything work?